rct229/ruleset_functions.py

```diff
@@ -0,0 +1,106 @@
+import rct229.rulesets as rulesets
+from rct229.rule_engine.rulesets import RuleSet
+from rct229.ruletest_engine.ruletest_jsons.scripts.excel_to_test_json_utilities import (
+    generate_rule_test_dictionary,
+)
+from rct229.schema.schema_enums import SchemaEnums
+from rct229.schema.schema_store import SchemaStore
+from rct229.utils.assertions import assert_
+
+
+def count_number_of_rules(ruleset_standard):
```

Comment:
These functions (```count_number_of_rules``` and ```count_number_of_ruletest_cases```) were moved to this file b/c if I imported these functions from the ```web_application.py``` file, there was a circular import error.